package.json

```diff
@@ -11,13 +11,13 @@
   },
   "scripts": {
     "test": "jest --env node --forceExit",
-    "process-dataset": "node process-dataset",
-    "upload-image": "node upload-dataset-image",
-    "release-dataset": "node release-dataset",
-    "prepublish-docs": "cd ./data-validation && make docs",
+    "process-dataset": "node bin/process-dataset",
+    "upload-image": "node bin/upload-dataset-image",
+    "release-dataset": "node bin/release-dataset",
+    "prepublish-docs": "cd bin && make docs",
```

Comment:
just wondering, why is Makefile in `bin/` ?  Isn't Makefile a package meta thing that should live down here next to package.json?

Reply:
yeah, that was my decision, I wasn't sure where to put it, but yeah we can put it up a level